imports/plugins/core/i18n/server/i18n/index.js

```diff
@@ -2,13 +2,15 @@ import { loadTranslations } from "/server/startup/i18n";
 
 // import ar from "./ar.json";
 // import bg from "./bg.json";
+// import cs from "./cs.json";
```

Comment:
what's going on in this file? Why add commented lines?

Reply:
@spencern this is the piece we were just discussing: re, might be nice to move to the registry, ie: `translations: ["ar", bg"], etc and the use the plugin loader to import the translations, rather than requiring this loading process in every plugin.

Reply:
When I was studying the structure of the reaction, I've realised that i18n didn't had all the languages on the comments, and as I saw the comments I though they weren't removed for some reason, so I've added them as a reminder to ask you: There are a bunch of i18n with commented imports, do they matter? Are you guys saving them with any purpose?